point_of_sale.rst

```diff
@@ -13,7 +13,6 @@ Point of Sale
    point_of_sale/payment
    point_of_sale/advanced
    point_of_sale/advanced_pricing_features
-   point_of_sale/analyze
```

Comment:
It looks to me like your moved this content in getting_started. If so -> redirect rule.

Reply:
You're right.